app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -182,3 +182,48 @@ table
       padding: 4px
       font-size: 0.8em
       color: #505050
+
+// tooltips with html support
+span.op_tooltip
```

Comment:
I think regarding naming things @myabc once posted this link: http://www.hagenburger.net/BLOG/Modular-CSS-Class-Names.html